ratis-common/src/main/java/org/apache/ratis/protocol/RaftGroupMemberId.java

```diff
@@ -33,6 +37,12 @@ public static RaftGroupMemberId valueOf(RaftPeerId peerId, RaftGroupId groupId)
   private final RaftGroupId groupId;
   private final String name;
 
+  private RaftGroupMemberId(RaftPeerId peerId, RaftGroupId groupId, int numMembers) {
+    this.peerId = Objects.requireNonNull(peerId, "peerId == null");
+    this.groupId = Objects.requireNonNull(groupId, "groupId == null");
+    this.name = "p:" + peerId + "@g:" + groupId + "@s:" + numMembers;
```

Comment:
Nit: should `@n` be used for `numMembers`? `n` means number. What `s` stands for?